edk2misc.c

```diff
@@ -0,0 +1,530 @@
+//
+//  edk2misc.c
+//  gfxutil
+//
+//  Created by joevt on 2019-12-14.
+//
+
+#include "edk2misc.h"
+
+//#include <stdio.h>
+
+// #undef __pic__
```

Comment:
Please remove commented out code here, above and below.

Reply:
Fixed.